runners/flink/job-server/flink_job_server.gradle

```diff
@@ -88,6 +88,7 @@ dependencies {
 //  TODO: Enable AWS and HDFS file system.
   // For resolving external transform requests
   runtime project(":sdks:java:io:kafka")
+  runtime project(":sdks:java:io:jdbc")
```

Comment:
I'm not sure there's actually any logic in the Python side that will ever expand jdbc from the expansion server built into Flink. I'd suggest removing this (unless I'm mistaken and there is some path on the Python side that will use it).

Reply:
Indeed it wasn't needed here.